miso-web/src/main/webapp/scripts/hot_library.js

```diff
@@ -292,6 +292,29 @@ HotTarget.library = (function() {
           HotUtils.makeColumnForText('Description', config.showDescription, 'description', {
             validator: HotUtils.validator.optionalTextNoSpecialChars
           }),
+          {
+            header: 'Date of creation',
+            data: 'creationDate',
+            type: 'date',
+            dateFormat: 'YYYY-MM-DD',
+            datePickerConfig: {
+              firstDay: 0,
+              numberOfMonths: 1
+            },
+            allowEmpty: true,
+            include: true,
```

Comment:
`!config.isLibraryReceipt`

Reply:
Hmm. I think it should be null as we don't necessarily have that information available (as the libraries would have been created at the "sending" lab, not in our lab).